src/server/passport.js

```diff
@@ -0,0 +1,114 @@
+const passport = require('passport');
```

Comment:
We don't do any auth in the ingest repo...therefore we don't need this, right?

Reply:
will pull it